Triangulation_on_sphere_2/doc/Triangulation_on_sphere_2/Triangulation_on_sphere_2.txt

```diff
@@ -0,0 +1,241 @@
+namespace CGAL {
+/*!
+
+\mainpage User Manual
+\anchor Chapter_2D_Triangulations_on_sphere
+
+\cgalAutoToc
+\author Mael Rouxel-Labbé, Monique Teillaud, and Claudia Werner
+
+\cgalFigureAnchor{fig_TOS2_header}
+<center>
+<img src="header.png" style="max-width:70%;"/>
+</center>
+
+This chapter describes two-dimensional triangulations on the sphere of \cgal. It is organized as follows:
+<ul>
+  <li>Section \ref Section_2D_ToS_Definitions introduces the main definitions about triangulations on the sphere.</li>
+  <li>Section \ref Section_2D_ToS_Implementation details the way two-dimensional triangulations on the sphere
+      are implemented and represented in \cgal.</li>
+  <li>Section \ref Section_2D_ToS_Examples lists a few basic examples to illustrate the capabilities
+      of the package.</li>
+</ul>
+
+\section Section_2D_ToS_Definitions Definitions
+
+Denote by \f$ \mathbb{S(c, r)}\f$ the two-dimensional sphere embedded in the Euclidean space \f$ \mathbb{R}^3\f$,
+with center `c` and radius `r`, that is
+\f$ \mathbb{S(c, r)} = \left\{ x \in \mathbf{R}^3 : \left\| x - c \right\| = r \right\} \f$.
+When the parameters `c` and `r` are not important, they will be omitted and \f$ \mathbb{S}\f$
+will be used directly.
+Given a set \f$ \mathcal{P}\f$ of points on \f$ \mathbb{S(c, r)}\f$, a <em>two-dimension triangulation</em>
+of \f$ \mathcal{P}\f$ can be described as a two-dimensional simplicial complex that is pure,
+connected, and without singularity whose vertices are exactly the points in \f$ \mathcal{P}\f$
+(see the complete definition in the package
+\link Section_2D_Triangulations_Definitions 2D Triangulations\endlink).
+
+In \f$ \mathbb{R}^2\f$, a <em>Delaunay</em> triangulation is a two-dimension triangulation
+that satisfies the <em>empty circle property</em> (also called <em>Delaunay property</em>):
+the circumscribing circle of any facet of the triangulation contains no point in its interior.
+This definition naturally extends to the two-dimensional sphere, as illustrated in the figure below.
+
+\cgalFigureBegin{fig_del_def, delaunay_on_sphere_def.png}
+Illustration of the Delaunay property: the circumscribing circle on the sphere (in green)
+of the Delaunay face \f$ p_1 p_2 p_3\f$ is empty.
+\cgalFigureEnd
+
+A particularity of Delaunay triangulations on \f$ \mathbb{S}\f$ is that the Delaunay property
+test can be reduced to a simple orientation test in the three-dimensional space: indeed, the circumscribing
+circle of a Delaunay face is the intersection of the supporting plane of the face with \f$ \mathbb{S}\f$,
+and determining whether a fourth point `q` is within the circle or not is equivalent to checking whether `q`
+is above, on, or below the supporting plane.
```

Comment:
you have within or not vs above,on, below.

Reply:
Within the face on the 2-manifold <=> above in 3D, that's intended; I am not sure what you mean?

Reply:
it's just that you compare `{within, not within}` to `{on, below, above}`